airbyte-webapp/src/pages/connections/types.ts

```diff
@@ -4,4 +4,5 @@ export const enum ConnectionRoutePaths {
   Transformation = "transformation",
   Replication = "replication",
   Settings = "settings",
+  JobHistory = "job-history",
```

Comment:
Just want to make sure that this name makes sense. The terminology is currently "syncs", and I wonder if this has been thought through roughly before changing the term to jobs.

Reply:
I think that's an @andyjih question

Reply:
Job History is correct!